scripts/storage-emulator-integration/import/tests.ts

```diff
@@ -0,0 +1,55 @@
+import * as path from "path";
```

Comment:
Just for completeness can you include a test that exports and then imports that data to verify data integrity

Reply:
Done.